src/js/components/molecules/in-page-notification/in-page-notification.scss

```diff
@@ -1,3 +1,4 @@
+// merge notification styling into here?
```

Comment:
Eh, wait what?

Reply:
There's `.notification` styling that we might be able to merge here